### PR TITLE
added repository class to prevent unneccesary extending

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1987,4 +1987,60 @@ and a your customjsonform fragment class initialize this interactor from createP
...
+>         "sort": true,
+>         "groupings": "[A,B,C,D,E,W,X,Y,Z]",
+>         "sortClass": "<fqn of sorting comparator class>",
+>         "type": "multi_select_list",
+>         "buttonText": "+ Add disease code",
+>         "dialogTitle": "Add disease code",
+>         "searchHint": "Type Disease Name"
+>       }
+
+    sort - default false, if true data will be sorted
```
Comment (dubdabasoduba) [inline README.md]: What kind of sort is it? i.e alphabetical e.t.c

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/MultiSelectListFactory.java [hunk 2]
```diff
@@ -166,12 +167,32 @@ private void prepareMultiSelectHashMap(@NonNull String stepName, boolean popup,
         if (StringUtils.isBlank(source)) {
             return MultiSelectListUtils.loadOptionsFromJsonForm(jsonObject);
         } else {
-            List<MultiSelectItem> fetchedMultiSelectItems = fetchData();
-            if(fetchedMultiSelectItems == null || fetchedMultiSelectItems.isEmpty()){
-                Utils.showToast(context, context.getString(R.string.multi_select_list_msg_data_source_invalid));
-                return null;
+            try {
+                String strRepositoryClass = jsonObject.optString(JsonFormConstants.MultiSelectUtils.REPOSITORY_CLASS);
+                Class<?> aClass = Class.forName(strRepositoryClass);
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/MultiSelectListFactory.java]: Please test this.

File: README.md [hunk 3]
```diff
@@ -1987,4 +1987,60 @@ and a your customjsonform fragment class initialize this interactor from createP
         .....
      }
  ````
+ 
+ ### MultiSelectList Widget
```
Comment (ekigamba) [inline README.md]: This is nice, if you were to use this the first time, would this information just get you started on it? 

Kindly see if you schedule time to add it to the native form sample which makes it easier to understand how the widget works and looks like

Comment (bennsimon) [reply README.md]: sample is there

**Pull request merged**